byterun/pyvm2.py

```diff
@@ -1033,11 +1172,59 @@ def byte_BUILD_CLASS(self):
     elif PY3:
         def byte_LOAD_BUILD_CLASS(self):
             # New in py3
-            self.push(__build_class__)
+            self.push(build_class)
 
         def byte_STORE_LOCALS(self):
             self.frame.f_locals = self.pop()
 
     if 0:   # Not in py2.7
         def byte_SET_LINENO(self, lineno):
             self.frame.f_lineno = lineno
+
+if PY3:
+    def build_class(func, name, *bases, **kwds):
```

Comment:
did we need to rewrite this function? I think the scope of this project is confined to the interpreter loop and function calling. I may have missed some new reason why this needs to be in Python now though.

Reply:
See the comment further down in the function, and my remarks on the pull request https://github.com/nedbat/byterun/pull/20 -- it might be better to discuss this over there instead. I'm not sure if any of your other comments are about my old submission instead of @vrthra's, since I've forgotten what was in it by now.

I vaguely remember trying alternatives, but couldn't find any better way to get Py3.4 supported.